### ISSUE TITLE
iOS Bottom Safe Area

### ISSUE THREAD

**Issue by okwasniewski** [okwasniewski/react-native-bottom-tabs#14]
When embedding a scroll view inside of a Tab Bottom Safe are of tab bar is not respected by React Native causing part of the scroll view to be covered by tab bar.

**Comment by dylancom** [comment]
Hmm I tried to solve this on the native side but failed.
Possible solution: expose useBottomTabBarHeight like in react-navigation.
```
const bottomTabBarHeight = useBottomTabBarHeight();

<ScrollView style={{ marginBottom: bottomTabBarHeight, overflow: "visible" }}  />
```

Not sure if we could come up with something smarter.

**Comment by dylancom** [comment]
In #31 I am experimenting with switching to UIKit and noticed that the scrollview respects the tabbar...

**Comment by okwasniewski** [comment]
> In #31 I am experimenting with switching to UIKit and noticed that the scrollview respects the tabbar...

SwiftUI uses the same component underneath so maybe we can make it work automatically

**Comment by okwasniewski** [comment]
Should be correctly addressed in the newly released version: 0.0.10
